x-pack/test/cases_api_integration/security_and_spaces/tests/common/configure/get_configure.ts

```diff
@@ -46,11 +47,22 @@ export default ({ getService }: FtrProviderContext): void => {
     });
 
     it('should return a configuration', async () => {
```

Comment:
super nit: usually it is better to have dedicated tests for new features like "it returns the custom fields correctly".